class.jetpack-data.php

```diff
@@ -93,22 +93,6 @@ public static function is_usable_domain( $domain, $extra = array() ) {
 			return new WP_Error( 'fail_subdomain_wpcom', sprintf( __( 'Domain `%1$s` just failed is_usable_domain check as it is a subdomain of WordPress.com.', 'jetpack' ), $domain ) );
 		}
 
-		// If PHP was compiled without support for the Filter module (very edge case)
-		if ( ! function_exists( 'filter_var' ) ) {
-			// Just pass back true for now, and let wpcom sort it out.
-			return true;
-		}
-
-		// Check the IP to make sure it's pingable.
-		$ip = gethostbyname( $domain );
-
-		// Doing this again as I was getting some false positives when gethostbyname() flaked out and returned the domain.
-		$ip = filter_var( $ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV4 ) ? $ip : gethostbyname( $ip );
-
-		if ( ! filter_var( $ip, FILTER_VALIDATE_IP, FILTER_FLAG_NO_PRIV_RANGE | FILTER_FLAG_NO_RES_RANGE | FILTER_FLAG_IPV4 ) && ! self::php_bug_66229_check( $ip ) ) {
-			return new WP_Error( 'fail_domain_bad_ip_range', sprintf( __( 'Domain `%1$s` just failed is_usable_domain check as its IP `%2$s` is either invalid, or in a reserved or private range.', 'jetpack' ), $domain, $ip ) );
-		}
-
```

Comment:
Why is all of this section removed? And alongside that... why did we ever need it in the first place? :)